Lib/glyphsLib/builder.py

```diff
@@ -463,6 +463,16 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
             value = int(value)
 
+        if name == 'Replace Feature':
+            if len(value.split(';')) > 2:
+                tag = value.split(';')[0]
+                code = ';'.join(value.split(';')[1:])
```

Comment:
if you only want to split at the first occurrence of a semicolon, you could do
```python
code = value.split(';', 1)
```
instead of splitting and re-joining